frontend/src/container/AppLayout/index.tsx

```diff
@@ -331,6 +385,58 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 					</Sentry.ErrorBoundary>
 				</div>
 			</Flex>
+
+			<div className="chat-support-gateway">
+				<Button
+					className="chat-support-gateway-btn"
+					onClick={(): void => setIsAddCreditCardModalOpen(true)}
```

Comment:
NIT : can we extract this out to a separate file ? we should keep the `AppLayout` lean.

Reply:
Moved ✅